corehq/apps/commtrack/management/commands/check_multiple_parentage.py

```diff
@@ -1,44 +0,0 @@
-from django.core.management.base import BaseCommand
```

Comment:
not a big deal, but in a PR like this if you're just deleting cruft I always prefer to just submit that separately